### DISCUSSION TITLE
posts/2024/03/life-song-4/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#43] [General]
# posts/2024/03/life-song-4/

https://guozheng.netlify.app/posts/2024/03/life-song-4/

**Comment by liechi** [comment]
看你在微信上被删，有点心情分享。我基本上不会主动加人微信；在别人希望加我时，一般也会委婉拒绝；也有勉强加了之后犹豫之下又删掉的情况。其实，拒绝和删除都不代表我不喜欢对方，因为我有不少没在微信上的朋友。我不知道别人发现删时，心情会怎么样，但推想起来大概是不好的。正如你说的，心里“来那么一下子”。所以除了少数例外，大多数情况下删微信的时候，心里会略感抱歉。

说起听歌，最近会单曲循环这个：https://www.bilibili.com/video/BV1SL4y1G7Fp/?spm_id_from=333.788&vd_source=c6250c6477208d8653dceb07ab4ba199

按你的年龄，可能没有看过这个电影。

你梦里的话确实看得人云里雾里。我的感觉是反过来：爱不是越界，而是做，在行动中付出自己。

**Reply by residualsun1** [reply]
> 其实，拒绝和删除都不代表我不喜欢对方

嗯。我有时候也会想过删除一些基本不会有交集的好友，但仅仅是因为想到我们之间再无联系的可能，甚至彼此之间也不熟悉。现在微信上流行一种做法，就是在朋友圈发一条说说，内容大概是相互点赞，相互删除，文案还挺客气有礼，或许也算作另类的告别吧。

> 按你的年龄，可能没有看过这个电影。

搜了一下，原来是05年的《[神话](https://zh.wikipedia.org/wiki/%E7%A5%9E%E8%AF%9D_(%E7%94%B5%E5%BD%B1))》，我确实没有看过。不过我个人觉得，这样的音乐配上画面/剧情后才“丰富”起来，不单纯是旋律，蒙毅的眼神、公主的笑容，这些配上音乐就吸引人了。

> 我的感觉是反过来：爱不是越界，而是做，在行动中付出自己。

好诠释: )。
